Solutions/Azure Web Application Firewall (WAF)/Analytic Rules/MaliciousWAFSessions.yaml

```diff
@@ -38,7 +38,8 @@ query: |
       | where TimeGenerated > ago(queryperiod)
       | where Category == 'ApplicationGatewayAccessLog' and (isempty(httpStatus_d) or httpStatus_d in (successCode))
```

Comment:
Do you think this comparison should also be case insensitive?

Reply:
Thanks @rahul0216 . This has been updated to support case-insensitivity.